app/main/views/auth.py

```diff
@@ -206,6 +228,11 @@ def confirm_dept():
 
     if form.validate_on_submit():
         if form.confirm.data == 'yes':
+
+            if current_app.config.get('DISABLE_INTERSTITIALS'):
```

Comment:
Am I right in thinking `DISABLE_INTERSTITIALS` short-circuits auth via the IdP? If so could we achieve this by mocking / spying in the tests rather than having this switching logic in the code proper?